pynest/nest/lib/hl_api_simulation.py

```diff
@@ -105,11 +109,15 @@ def Run(t):
 
 @check_stack
 def Prepare():
-    """Calibrate the system before a `Run` call. Not needed for `Simulate`.
+    """Calibrate the system before a `Run` call.
+    `Prepare` is automatically triggered at beginning of call to `Simulate`:
+
+    Simulate(t): t' = t/m; Prepare(); for _ in range(m): Run(t'); Cleanup()
 
     Call before the first `Run` call, or before calling `Run` after changing
     the system, calling `SetStatus` or `Cleanup`.
 
+
```

Comment:
Here and below: is this intentional?

There is only a single blank line above titles elsewhere.
```suggestion
```

Reply:
No this is unintentional, I fixed this and also I noticed there should be a new line after the title of the docstring.